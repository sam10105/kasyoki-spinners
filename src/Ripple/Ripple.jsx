import React from 'react';
import './style.css';

const Ripple = () => (
  <div className="ripple">
    <div />
    <div />
  </div>
);

export default Ripple;
