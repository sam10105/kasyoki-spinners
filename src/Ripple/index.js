export { default as Ripple } from './Roller';
