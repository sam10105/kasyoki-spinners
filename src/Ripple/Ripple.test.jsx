import React from 'react';
import { render } from '@testing-library/react';
import Ripple from './Ripple';

it('should take a snapshot', () => {
  const { asFragment } = render(<Ripple />);

  expect(asFragment(<Ripple />)).toMatchSnapshot();
});
