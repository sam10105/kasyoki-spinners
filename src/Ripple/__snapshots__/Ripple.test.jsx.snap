// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="ripple"
  >
    <div />
    <div />
  </div>
</DocumentFragment>
`;
